src/syrupy/report.py

```diff
@@ -89,7 +134,7 @@ def num_unused(self) -> int:
 
     @property
     def ran_all_collected_tests(self) -> bool:
-        return self.all_items == self.ran_items and not self.is_providing_nodes
+        return self.all_items == self.ran_items
```

Comment:
Moved the ` and not self.is_providing_nodes` check to the point of usage at `if self.ran_all_collected_tests and not any(provided_nodes):`, makes the method name more accurate.